LeetCodeQuestions.md

```diff
@@ -0,0 +1,126 @@
+
+## Array
+
+- [x] [Two Sum](https://leetcode.com/problems/two-sum/)
```

Comment:
What does the check mean?

Reply:
I had grabbed the list from the Blind 75 webpage and I assume it was how far along the creator is, I updated the list with my progress